### PR TITLE
Finished Arch and Fedora Sections 

### LINKED ISSUE #1: Fedora Review

### LINKED ISSUE #5: Arch Review

### PR DESCRIPTION
**Pull request by thisisamna**
Finished Arch and Fedora sections for LDP 2024, closes #1 , closes #5

### CODE REVIEW

File: Arch/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+![](imgs/getstart.img.png)
+
+* choose the first option
+
+![](imgs/firstpage.img.png)
+
+* then this page will appear 
+
+![](imgs/secondpage.img.png)
```
Comment (Dimitry-bit) [inline Arch/README.md]: The installation guide is uninformative and provides no real value to the user; it is better to just remove it.

File: Fedora/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
+# Fedora
+![Fedora_LOGO](Imgs/Fedora_LOGO.png)
```
Comment (Dimitry-bit) [inline Fedora/README.md]: The logo is huge. Set it to a reasonable size.

File: Fedora/README.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+1. Write your name in the **Full Name** field and click on **Next**.![VB Step 27](<Imgs/VB Step27.png>)
+
+1. Write a strong password for your system in **Password** field and confirm it then click on **Next**.![VB Step 28](<Imgs/VB Step28.png>)
+
+1. Click on **Start Using Fedora Linux**.![VB Step 29](<Imgs/VB Step28.png>)
+
+1. Enjoy using your system :)
+
+All these steps are based on this [tutorial](https://youtu.be/4eJ3pF4-CN4?si=2ydu6BxTvOByQWJv).
+
```
Comment (Dimitry-bit) [inline Fedora/README.md]: The screenshots clearly display Windows' taskbar, which is unacceptable.

File: Fedora/README.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+1. Choose the hard drive that you want Fedora to be installed on, once finished, Click done.![Dual Boot Step8](<Imgs/Dual Boot Step8.png>)
+
+1. You will be redirected to screen mentioned in step 7, Click begin installation and wait for it to finsh.
+
+1. Reboot your device and remove USB stick.
+
+1. You will then be prompted with this screen to continue setting up your user info and to edit privacy and enable third party repositories.![Dual Boot Step11](<Imgs/Dual Boot Step11.png>)
+
+1. Enjoy using Fedora!
```
Comment (Dimitry-bit) [inline Fedora/README.md]: This installation guide is labeled for dual boot but fails to mention any dual boot related information throughout the guide.

**Review by Dimitry-bit — CHANGES_REQUESTED** [review]
Both reviews are quite rough and require more polish. Please fix the following issues and re-request a review.

File: Fedora/README.md (hunk 4 above)
Comment (thisisamna) [reply Fedora/README.md]: On it! Appreciate the feedback

File: Fedora/README.md [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+- Flatpak is installed by default on Fedora.
+
+---
+
+- Easy to use, simple, and beginner friendly.
+- 100% free.
+- Good balance between bleeding edge and stability.
+- Customizable.
+- Flatpak is installed by default on Fedora.
+
```
Comment (Dimitry-bit) [inline Fedora/README.md]: Fix duplication.

File: Fedora/README.md [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+1. Go to **Settings => System => Display** and in the **Video memory** section make it 128 MBs. ![VB Step 11](<Imgs/VB Step11.png>)
+
+1. Go to **Settings => System => Shared folders** then Click on **+** button in the upper right corner and choose **Other**.
+1. Go to **Settings => System => Shared folders** then Click on **+** button in the upper right corner and choose **Other**.
+
+1. In the **Folder Path** section you can share any local drive you want and check **Auto-Mount** option then press **OK => OK** (steps 12 & 13 are optional).![=VB Step 13](<Imgs/VB Step13.png>)
+
+1. Click on **start** button and wait for a while.
+
```
Comment (Dimitry-bit) [inline Fedora/README.md]: More duplication.

**Review by Dimitry-bit — CHANGES_REQUESTED** [review]
Great work, but some duplication exists.

**Review by Dimitry-bit — APPROVED** [review]
Nice work!

**Comment by thisisamna** [comment]
Appreciate the reviews and feedback!

**Pull request merged**